build/Sdk.slnf

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This change was not in the [original SDK refactor PR](https://github.com/Azure/azure-functions-dotnet-worker/pull/1946/files). Do we need this?

Reply:
This is part of the refactor to our test steps. 

it is used in `TestUtility.cs`
``` CSharp
public static readonly string SdkBuildProj = Path.Combine(PathToRepoRoot, "build", "Sdk.slnf");
```